composer.json

```
@@ -4,12 +4,12 @@
     "type": "ezplatform-bundle",
     "license": "GPL-2.0-only",
     "require": {
-        "php": "^7.1",
-        "symfony/http-kernel": "^3.4",
-        "symfony/yaml": "^3.4",
+        "php": "^7.3",
```

The others changed for 7.2, and with both Ubuntu LTS and the newly release RHEL 8 [using 7.2](https://doc.ezplatform.com/en/latest/getting_started/requirements/) that would be strongly preferable.
```suggestion
        "php": "^7.2",
```

---

> With both current Ubuntu LTS and the newly release RHEL 8 [using 7.2](https://doc.ezplatform.com/en/latest/getting_started/requirements/) that would be strongly preferable as the new minimum.

I'm running Ubuntu 18.04 LTS and had no problem with installing PHP 7.3. The instruction for that is widely available and pretty straightforward. If we want to have some future 3.x LTS release, we should go for the highest possible version.

---

Yes I second what Andrew said. I'd say we have to be on the highest possible PHP version available right now because there is going to be an LTS release in 3.x. I bet it's already possible to install PHP 7.3 on REHL/CentOS so why would we care? It's just a matter of upgrading one package in the OS.

---

Valid arguments, but:
1. Our _customers_ are not developers.
Yes you can install PHP 7.3 on any OS out there, from source or thirdparty. But often we face IT departments or business owners who reject solutions if they can't use official packages they pay for _(under SLA / support contract)_. 

2. As this is potentially a beyond eZ package, it ideally should align with Symfony 4 (PHP 7.1.3) if the intention is for it to gain ground in the wider symfony community.

However for customers we can offer then 2.5LTS instead for the next 2 years, so I'm _not_ saying this is critical, just not optimal for sales. Customers will easily be able to upgrade to 3.x later once they are ready for that right? 😏